railties/lib/rails/generators/testing/assertions.rb

```diff
@@ -121,6 +121,26 @@ def assert_field_default_value(attribute_type, value)
             assert_equal(value, create_generated_attribute(attribute_type).default)
           end
         end
+
+        # Asserts a given initializer exists. You need to supply a path relative
+        # to the `config/initializers/` directory.
+        #
+        #   assert_initializer "mail_interceptors.rb"
+        #
+        # You can also give extra arguments. If the argument is a regexp, it will check if the
+        # regular expression matches the given file content. If it's a string, it compares the
+        # file with the given string:
+        #
+        #   assert_initializer "mail_interceptors.rb", /SandboxEmailInterceptor/
+        #
+        # Finally, when a block is given, it yields the file content:
```

Comment:
I don't quite see a block argument usage below.

Reply:
True. And since there isn't any test for it, we didn't caught. I'm fixing it

Reply:
There could be value in keeping it for cases like this:

```ruby
expcted = <<~RUBY
  Rails.application.configure do
    if Rails.env.staging?
      config.action_mailer.interceptors = %w[SandboxEmailInterceptor]
    end
  end
RUBY

assert_initializer "mail_interceptors.rb" do |initializer|
  assert_equal expcted, initializer
end
```

Reply:
I kept it.